### PR TITLE
Add cancellations table

### PR DESCRIPTION
**Pull request by porras**
Required by travis-pro/billing#230.

Note: `structure.sql` includes a couple of unrelated changes because it hadn't been updated in some previous pull request, apparently.

### CODE REVIEW

File: db/main/migrate/20180725000000_create_subscription_cancelations.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreateSubscriptionCancelations < ActiveRecord::Migration[4.2]
```
Comment (carlad) [inline db/main/migrate/20180725000000_create_subscription_cancelations.rb]: Cancellation is spelt incorrectly. Should be `CreateSubscriptionCancellations`

File: db/main/migrate/20180725000000_create_subscription_cancelations.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  def change
+    create_table :subscription_cancelations do |t|
```
Comment (carlad) [inline db/main/migrate/20180725000000_create_subscription_cancelations.rb]: `:subscription_cancellations`

**Comment by porras** [comment]
@carlad thanks for the timely review! I fixed those things, I also took the opportunity to remove the `subscriptions_` preffix, which was more noise than anything else.

File: db/main/migrate/20180725000000_create_cancellations.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class CreateCancellations < ActiveRecord::Migration[4.2]
+  def change
+    create_table :cancellations do |t|
+      t.references :subscription, index: true, null: false
+      t.references :user, null: false
+      t.string :plan, null: false
+      t.date :subscription_start_date, :cancelation_date, null: false
```
Comment (carlad) [inline db/main/migrate/20180725000000_create_cancellations.rb]: another `cancellation` - sorry i missed this one...

**Pull request merged**